toast-tk-rest-utils/src/main/java/io/toast/tk/core/rest/InspectPage.java

```diff
@@ -9,18 +9,32 @@
 public class InspectPage {
 
 	public String name;
+	
+	public Double erreur;
 
 	public List<String> items = new ArrayList<>();
 
 	protected InspectPage() {
-
+		this.erreur = null;
 	}
 
 	public InspectPage(
 		final String item,
 		final List<Object> items
 	) {
+		this.erreur = null;
 		this.name = item;
 		items.stream().forEach(o -> items.add(o.toString()));
 	}
+	
+	private void codepouri() {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "codepouri" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)